.github/workflows/release.yml

```diff
@@ -1,11 +1,53 @@
-name: Publish Release
+name: Test and publish a new release
   
 on:
-  release:
-    types: [published]
-    branches: [master]
+  push:
+    tags:
+      - 'v*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SBT_CREDENTIALS: '.credentials'
   
 jobs:
-  release:
-    uses: evolution-gaming/scala-github-actions/.github/workflows/release.yml@v1
```

Comment:
would you suggest/consider to move all our OSS libraries to this approach by creating `release.yml@v2` action?

Reply:
sure, if it works well and i find no issues with it then i don't see why not